VRDR.Client/Client.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System;
+using System.Text;
+using System.Threading.Tasks;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using Newtonsoft.Json.Linq;
+using VRDR;
+
+
+namespace nvssclient.lib;
+public class Client
+{
+    /// <summary>The API url</summary>
+    public String Url { get; }
+    /// <summary>Whether the client is running locally</summary>
+    public bool LocalTesting { get; }
+    /// <summary>The credentials to access the API server</summary>
+    public Credentials Credentials { get; }
+    /// <summary>The token to access the API server</summary>
+    public string Token { get; set; }

```

Comment:
Token gets checked for null/empty a few times, and its value gets used for the authentication header, but its value never gets set anywhere. How is it supposed to get a value assigned?